app/actions/deployment_create.rb

```diff
@@ -201,15 +203,23 @@ def supersede_deployment(previous_deployment)
         )
       end
 
-      def log_rollback_event(app_guid, user_id, revision_id)
+      def starting_state(message)
+        if message.strategy == DeploymentModel::CANARY_STRATEGY
+          DeploymentModel::PREPAUSED_STATE
+        else
+          DeploymentModel::DEPLOYING_STATE
+        end
+      end
+
+      def log_rollback_event(app_guid, user_id, revision_id, strategy)
         TelemetryLogger.v3_emit(
           'rolled-back-app',
           {
             'app-id' => app_guid,
             'user-id' => user_id,
             'revision-id' => revision_id
           },
-          { 'strategy' => 'rolling' }
+          { 'strategy' => strategy }
```

Comment:
Asking cause it's faster than me hunting... 😛 Is there a telemetry event for creating a canary deployment?